Traceback (most recent call last):
  File "cavendish_balance.py", line 13, in <module>
    times =np.loadtxt(str(sys.argv[2]), dtype='float')
  File "/usr/lib/python2.7/dist-packages/numpy/lib/npyio.py", line 896, in loadtxt
    fh = iter(open(fname, 'U'))
IOError: [Errno 2] No such file or directory: 'time_tiral1.dat.dat'
